src/Controls/src/Core/HandlerImpl/Window/Window.Impl.cs

```diff
@@ -468,10 +468,6 @@ void OnPopCanceled()
 
 		void IWindow.Created()
 		{
-#if DEBUG
-			if (IsCreated)
-				throw new InvalidOperationException("Window was already created");
```

Comment:
Removing these for now so we don't create weirdness with backporting. 

This [PR](https://github.com/dotnet/maui/pull/13025) will just throw on release and debug so things don't break in odd ways.